### ISSUE TITLE
Using UKF

### ISSUE THREAD

**Issue by prozoroff** [marioslokas/KalmanFilters#1]
Hello,
I like your UKF code
Are you using 
http://www.mathworks.com/matlabcentral/fileexchange/18217-learning-the-unscented-kalman-filter/content/ukf.m
to translate logic to C# ?
Could you provide some example of using this algorithm.

**Comment by marioslokas** [comment]
Yes, this is the logic of Yi Cao Matlab implementation for C#. I also used stuff from EmguCV Kalman filter example. I added a small readme about the code and will update to a better version when I can. If you still don't understand let me know.

**Comment by prozoroff** [comment]
Thanks a lot!

**Comment by lbarwick** [comment]
Mario, I am using prozoroff UKF code, which has same root as yours. We are trying to adapt it to have multiple timeseries of prices from FOREX and attempt to predict value of a single price.
I would love to be able to discuss this work with you. If you are interested please email me at xmoon2000[at]googlemail[dot]com
